CMakeLists.txt

```diff
@@ -294,7 +295,7 @@ endif()
 find_package(Armadillo "${ARMADILLO_VERSION}" REQUIRED)
 
 # Include directories for the previous dependencies.
-set(MLPACK_INCLUDE_DIRS ${MLPACK_INCLUDE_DIRS} ${ARMADILLO_INCLUDE_DIRS})
+set(MLPACK_INCLUDE_DIRS ${MLPACK_INCLUDE_DIRS} ${ARMADILLO_INCLUDE_DIRS} ${CEREAL_INCLUDE_DIR})
```

Comment:
What if we add `${CEREAL_INCLUDE_DIR}` after finding cereal(`find_package(cereal)`).